.circleci/build.sh

```diff
@@ -11,22 +10,6 @@ clone_pytorch $PYTORCH_DIR $XLA_DIR
 # Use bazel cache
 USE_CACHE=1
 
-SCCACHE="$(which sccache)"
```

Comment:
Let's leave this as-is, this is used in the upstream (nice to have the option to turn it on and off).

Reply:
Check this out, and leave this declaration as well https://github.com/pytorch/xla/blob/ea5073b4c445f14b1a6ecdfa86101b0d7da68a0d/.github/workflows/_build.yml#L89C45-L89C59

Reply:
hmm, I thought upstream also migrated to use our bazel remote cache, we even share the service key with them for this purpose.

Reply:
Will preserve sccache as it might be used in upstream and we may re-enable it in the future

Reply:
@JackCaoG not for pytorch build, i think. It's a few lines of code, so might as well leave as an option?